summaries/sprint4.md

```diff
@@ -0,0 +1,21 @@
+# Week 4 sprint retrospective
+
+## Bettens, Louis
+
+
+## Jaakik, Marouane
+
+
+## Konrad, Jonas Franz
+
+
+## Lachat, Niels Marco
+
+
+## Lenweiter, Martin
+
+
+## Vial, Nicolas
+
+
```

Comment:
```suggestion
This week I made the link between all the UIs that are part of the management of a dataset. I also made some modifications to use the model to communicate with the database.Making the link and all the modifications took me a lot of time because there were a lot of small details about what we wanted to be possible when using the application and details about the model that allows the UIs to communicate with the database. I didn't encounter any big problems during the implementation but I had some issues with some tests that I was able to fix after discussing with some team members.  I clearly worked a lot more than usual (about 20 hours) but since we decided to do a 1.5 week sprint, it didn't cause any big problems.
```